main/options/options.cc

```diff
@@ -314,9 +314,12 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
                                     cxxopts::value<string>()->default_value(empty.webTraceFile), "file");
     options.add_options("advanced")("debug-log-file", "Path to debug log file",
                                     cxxopts::value<string>()->default_value(empty.debugLogFile), "file");
-    options.add_options("advanced")("reserve-mem-kb",
-                                    "Preallocate the specified amount of memory for symbol+name tables",
-                                    cxxopts::value<u8>()->default_value(fmt::format("{}", empty.reserveMemKiB)));
+    options.add_options("advanced")(
+        "preallocate-symbol-size", "Preallocate the specified number of entries in the symbol table",
```

Comment:
My vote is something like `reserve-symbol-table-capacity` because it's using `vector::reserve` to modify the `vector::capacity`.

Reply:
That's fine with me.